### PR TITLE
feature/206

### PR DESCRIPTION
**Pull request by VB10**
- Remove unused imports and files
- Update project name and imports
- Update import statements in multiple files
- Update dependencies and translations, add new icons and export new button widgets

### CODE REVIEW

**Comment by VB10** [comment]
latest--
Screenshot 2024-02-25 at 23 09 51

File: pubspec.yaml [hunk 1]
```diff
@@ -91,8 +91,7 @@ dependencies:
   json_annotation: ^4.8.0
 
   life_shared:
-    git:
-      url: https://github.com/VB-CORE/life_shared.git
+      path: /Users/velibacik/Desktop/development/life/life_shared
```
Comment (mysCod3r) [inline pubspec.yaml]: hocam path local kalmış

**Review by mysCod3r — COMMENTED** [review]
Elinize sağlık hocam

File: pubspec.yaml (hunk 1 above)
Comment (VB10) [reply pubspec.yaml]: Yesss düzeliyorum

**Pull request merged**